addons/knobs/README.md

```diff
@@ -231,6 +231,18 @@ const value = date(label, defaultValue);
 
 > Note: the default value must not change - e.g., do not do `date('Label', new Date())` or `date('Label')`
 
+### button
+
+Allows you to include a button and associated handler.
+
+```js
+import { button } from '@storybook/addon-knobs';
+
+const label = 'Do Something';
+const handler = () => doSomething('foobar');
+const value = button(label, handler);
```

Comment:
What can you do with that `value`?

Reply:
Addressed, removed the assignment as it was unnecessary.